docs/docs/PythonUserGuide/run-from-pip.md

```diff
@@ -7,6 +7,21 @@
  * Only __Python 2.7__, __Python 3.5__ and __Python 3.6__ are supported for now.
  * Note that __Python 3.6__ is only compatible with Spark 1.6.4, 2.0.3, 2.1.1 and 2.2.0. See [this issue](https://issues.apache.org/jira/browse/SPARK-19019) for more discussion.
 
+
+## **Run as local program**
+ * If you use `LocalOptimizer`, the model would be train or validate in local which only base on Ndarray without using RDD or DataFrame.
+ * Check [LeNet](../../../../pyspark/bigdl/models/local_lenet/README.md) for more details
+
+ ```
+    # X_train, Y_train, X_test are all ndarray and the first dimension is the sample number.
+    local_optimizer = LocalOptimizer(
```

Comment:
Instead of explicitly specifying LocalOptimizer, can we have Optimizer to check if a local or distributed optimizer should be used at run time?